README.md

```diff
@@ -28,7 +28,7 @@ use dotenvy::dotenv;
 use std::env;
 
 fn main() {
-    dotenv().ok();
+    dotenv().unwrap();
```

Comment:
The [Rust API Guidelines](https://rust-lang.github.io/api-guidelines/documentation.html?highlight=unwrap#examples-use--not-try-not-unwrap-c-question-mark) advice using the question mark operator over unwrapping errors. Perhaps this would be more appropriate here as it is often the first example a new user will see.

Reply:
@xiaoas I know I said `unwrap()` in the issue,  but I actually think `expect()` is more appropriate here. Sorry for not being more clear.

Could you also add a comment above the `dotenv()` call? It would be helpful to explain why `dotenv()` is being called in the first place.

Reply:
done